.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
IMO editor/dev-specific code belongs in either your global `.gitconfig` or `.git/info/excludes`, `.gitignore` should be used for language/repo-specific files

(this is a very grumpy/pedantic opinion so feel free to ignore)